extensions/hibernate-orm/runtime/src/main/java/io/quarkus/hibernate/orm/runtime/PersistenceUnitsHolder.java

```diff
@@ -1,10 +1,6 @@
 package io.quarkus.hibernate.orm.runtime;
 
-import java.util.Collection;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.stream.Collectors;
+import java.util.*;
```

Comment:
Please don't use star imports (and ideally disable it in your IDE!).

I tried to fix it myself but you don't give the permission to push to your branch so you will have to do it yourself :)